packages/front-end/components/Layout/ProjectSelector.tsx

```diff
@@ -82,7 +82,7 @@ export default function ProjectSelector() {
             bold={!project}
           />
         </DropdownLink>
-        {projects.map((p) => {
+        {projects.sort((a, b) => a.name > b.name ? 1 : -1).map((p) => {
```

Comment:
`sort` modifies the original `projects` array which is used throughout the app, so that's why it's also affecting the settings page.

To fix this, you need to first make a shallow copy of the `projects` array and then sort it.  The two easiest ways to do this are using `slice` and the spread operator.

Either of these will work:
1. `projects.slice().sort(`
2. `[...projects].sort(`

Reply:
Awesome, thank you. Yeah I know when to use a shallow copy versus a deep copy in algorithms, I didn't think about it here though. I committed the change now. I ran the tests and linting both times too.